controllers/availabilityController.ts

```diff
@@ -0,0 +1,34 @@
+import { PrismaClient } from ".prisma/client";
+import express, { Request, Response, Router } from "express";
+import AvailabilityRepository from "../models/AvailabilityRepository";
+import { formatDistance, subDays } from "date-fns";
```

Comment:
We can add formatISO to this import like `{ formatDistance, subDays, formatISO }` as well.